### PR TITLE
Set screenmode for textmode installation

### PR DESCRIPTION
**Pull request by imobachgs**
Related to https://github.com/libyui/libyui-qt/pull/60 and https://github.com/libyui/libyui-qt/pull/61 but for textmode installation.

We have several themes for libyui-ncurses and one of them is called `highcontrast`.

### CODE REVIEW

File: package/yast2.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Oct 13 14:11:31 UTC 2016 - igonzalezsosa@suse.com
+
+- Set ncurses theme when Y2COLORMODE is specified at boot time
```
Comment (ancorgs) [inline package/yast2.changes]: Y2COLORMODE?

Comment (imobachgs) [inline package/yast2.changes]: Oh, too many renames :) Fixed now.

File: package/yast2.spec [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2
-Version:        3.1.208
+Version:        3.1.209
```
Comment (ancorgs) [inline package/yast2.spec]: Shouldn't we jump to 3.2?

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**